lib/guardian.rb

```diff
@@ -443,6 +443,15 @@ def can_send_private_message?(target, notify_moderators: false)
     (!is_silenced? || target.staff?)
   end
 
+  def can_send_private_message_generally?(notify_moderators: false)
```

Comment:
Could we use the existing `can_send_private_message?` method and check against the system user? We do something similar in the user card serializer: 

https://github.com/discourse/discourse/blob/master/app/serializers/user_card_serializer.rb#L132-L135

Reply:
For some reason yesterday this was really inconsistently working for me but today it is fine...will just get rid of the new method :S